cli/build.rs

```diff
@@ -440,11 +544,13 @@ fn main() {
   let c = PathBuf::from(env::var_os("CARGO_MANIFEST_DIR").unwrap());
   let o = PathBuf::from(env::var_os("OUT_DIR").unwrap());
 
-  // Main snapshot
   let compiler_snapshot_path = o.join("COMPILER_SNAPSHOT.bin");
-
   let js_files = get_js_files("tsc");
-  create_compiler_snapshot(&compiler_snapshot_path, js_files, &c);
+  ts::create_compiler_snapshot(&compiler_snapshot_path, js_files, &c);
+
+  let cli_snapshot_path = o.join("CLI_SNAPSHOT.bin");
+  let js_files = get_js_files("js");
```

Comment:
I would expect the new `cli/js` files to be included through a simple extension added to [`cli_exts()`](https://github.com/denoland/deno/blob/89c5aa85984eabd300818f8275e3970f12a89754/cli/ops/mod.rs#L14), would that avoid a lot of these changes?